plexpy/datafactory.py

```diff
@@ -1168,10 +1180,13 @@ def get_watch_time_stats(self, rating_key=None, grouping=None, query_days=None):
                                 'FROM session_history ' \
                                 'JOIN session_history_metadata ON session_history_metadata.id = session_history.id ' \
                                 'WHERE stopped >= %s ' \
-                                'AND (session_history.grandparent_rating_key = ? ' \
-                                'OR session_history.parent_rating_key = ? ' \
-                                'OR session_history.rating_key = ?)' % (group_by, timestamp_query)
-                        result = monitor_db.select(query, args=[rating_key, rating_key, rating_key])
+                                'AND (session_history.grandparent_rating_key IN (%s) ' \
+                                'OR session_history.parent_rating_key IN (%s) ' \
+                                'OR session_history.rating_key IN (%s))' % (
+                                    group_by, timestamp_query, rating_keys_arg, rating_keys_arg, rating_keys_arg
+                                )
+                        
+                        result = monitor_db.select(query, args=rating_keys*3)
```

Comment:
Put `timestamp_query` into the query args.
```suggestion
                                'WHERE stopped >= ? ' \
                                'AND (session_history.grandparent_rating_key IN (%s) ' \
                                'OR session_history.parent_rating_key IN (%s) ' \
                                'OR session_history.rating_key IN (%s))' % (
                                    group_by, rating_keys_arg, rating_keys_arg, rating_keys_arg
                                )
                        
                        result = monitor_db.select(query, args=[timestamp_query] + rating_keys * 3)
```

Reply:
Good to know the way with +.